jsonpatcher/jsonpatcher.go

```diff
@@ -202,4 +227,8 @@ func (je patchEvent) Type() string {
 	return je.TypeName
 }
 
+func (je patchEvent) Node() (format.Node, error) {
+	return cbornode.WrapObject(je, mh.SHA2_256, -1)
```

Comment:
the json patch struct is encoded in cbor to get the node that will be stored in the log.

Reply:
Niiiiice